Content/CustomHooks/Content.PrehardmodeWarning.cs

```diff
@@ -0,0 +1,26 @@
+﻿using StarlightRiver.Core.Loaders.UILoading;
+using StarlightRiver.Content.GUI;
+using StarlightRiver.Core.Systems.CameraSystem;
+using System;
+using System.Linq;
+using Terraria.DataStructures;
+using Terraria.ID;
+using Terraria.Localization;
+
+namespace StarlightRiver.Content.CustomHooks
+{
+	class PrehardmodeWarning : HookGroup
+	{
+		//Swaps the vanilla meteor events out, could create conflicts if other mods attempt the same but shouldnt be anything fatal
```

Comment:
Looks like this comment got left over from copy/pasting, might want to remove this or update it for the actual functionality of this class.